editor.planx.uk/src/components/Error/ErrorFallback.tsx

```diff
@@ -1,12 +1,15 @@
 import Typography from "@mui/material/Typography";
 import Card from "@planx/components/shared/Preview/Card";
 import { ErrorSummaryContainer } from "@planx/components/shared/Preview/ErrorSummaryContainer";
+import { logger } from "airbrake";
 import React from "react";
 
-import { logger } from "../airbrake";
+import { GraphErrorComponent, isGraphError } from "./GraphError";
 
-function ErrorFallback(props: { error: Error }) {
-  logger.notify(props.error);
+const ErrorFallback: React.FC<{ error: Error }> = ({ error }) => {
+  if (isGraphError(error)) return <GraphErrorComponent error={error} />;
```

Comment:
This is a neat solution & I think we'll appreciate not having extra noise from these, thanks!